rpc/dcrd.go

```diff
@@ -140,6 +141,26 @@ func (c *DcrdRPC) SendRawTransaction(txHex string) error {
 	return nil
 }
 
+// DCP0010Active uses getblockchaininfo RPC to determine if the DCP-0010 agenda
+// has activated on the current network.
+func (c *DcrdRPC) DCP0010Active() (bool, error) {
```

Comment:
Wouldn't `DCP0010Active` be a better name here?

Reply:
Umm... that is the name?

Reply:
Whoops, meant `IsDCP0010Active`.